### ISSUE TITLE
Add Github action to autoclose issues listed in the PR comments.

### ISSUE THREAD

**Issue by sbtaylor15** [ortelius/ortelius#201]
Need to add a 3rd party action for the autoclose.  The default github close only works for issues in the same repo.  Our issues are across repos.  

Once we get this working with one repo we will need to copy the action into the other repos.  Templating actions/workflows are not supported by GitHub. 

May want to look at a different CI/CD tool that handles templating or events, ie Jenkins Templating Engine, Keplar, Tekton

**Comment by sbtaylor15** [comment]
Added GitHub Action to auto close issue when PR is merged.  See [.github/workflows/autoclose.yaml](https://github.com/ortelius/ortelius/blob/maintenance/.github/workflows/autoclose.yml)


Format of close statements:
```
Fixes #1,#2,#3
close #1, #2, #3
fix #1 #2 #3
resolve #1,#2 #3
Resolves: #1,#2,#3
closed : #1, #2, #3

Fix https://github.com/ortelius/ortelius-docs/issues/16
```

Use full url if the issue exists in another repo.

Action needs to be copied to all repos.

**Comment by sbtaylor15** [comment]
Found that the action needs to be updated to handle link to issue.  Parsing is skipping over issues from another repo
